core/src/Pos/Core/Ssc/CommitmentAndOpening.hs

```diff
@@ -0,0 +1,37 @@
+module Pos.Core.Ssc.CommitmentAndOpening
+       ( randCommitmentAndOpening
+       ) where
+
+import           Universum
+
+import qualified Crypto.Random as Rand
+import qualified Data.HashMap.Strict as HM
+import qualified Data.List.NonEmpty as NE
+import           Formatting (build, sformat, (%))
+
+import           Pos.Binary.Class (asBinary)
+import           Pos.Core.Ssc.Commitment (Commitment (..))
+import           Pos.Core.Ssc.Opening (Opening (..))
+import           Pos.Crypto (Threshold, VssPublicKey, genSharedSecret)
+
+
+-- | Generate random SharedSeed.
+randCommitmentAndOpening
```

Comment:
Moved from https://github.com/input-output-hk/cardano-sl/pull/3089/files#diff-8ead799787158d7a4cfab05f0372d2a1L82.

Since our convention is to use `gen$FOO` for Hedgehog generators, I opted for `rand$FOO` since this is in MonadRandom.

Reply:
I much prefer this.

Reply:
🎉